### PR TITLE
ENH: Created a new data loader for Zacks data from Quandl

### PR DESCRIPTION
**Pull request by pbharrin**
Created a new data loader.

### CODE REVIEW

**Comment by FreddieV4** [comment]
Hey @pbharrin thanks for opening up this PR. It looks like the flake8 tests failed on CI due to these reasons:

image

**Comment by pbharrin** [comment]
I fixed those code style issues, and everything passes now.

File: zipline/data/bundles/zacks_quandl.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+        df = df.ix[START_DATE:]  # drop dates before START_DATE
+        gc.collect()  # force garbage collection to free up memory
+
+        # drop row with NaNs or the loader will turn all columns to NaNs
+        df = df.dropna()
+
+        uv = df.m_ticker.unique()  # get unique m_tickers (Zacks primary key)
+
+        # counter of valid securites, this will be our primary key
```
Comment (Gaffney) [inline zipline/data/bundles/zacks_quandl.py]: nit: securites  -> securities

File: tests/data/bundles/test_zacks.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+from zipline.testing.predicates import (
+    assert_equal,
+)
+
+
+class ZacksBundleTestCase(ZiplineTestCase):
+    """
+    Class for testing the Zacks daily data bundle.
+    An test file is stored in tests/resources/zacks_samples/fictious.csv
+
```
Comment (Gaffney) [inline tests/data/bundles/test_zacks.py]: nit: rm

**Comment by ernestoeperez88** [comment]
It looks like conversation in this PR has gone stale. As mentioned above, this would be better suited as an extension to zipline. Please feel free to reach out if you have any questions.

**Pull request closed without merging**